### PR TITLE
feat(web): Life Events UI 2.0 - Homepage

### PR DESCRIPTION
**Pull request by albinagu**
Life Events Sections gets a new look on the frontpage:
Screenshot 2024-03-04 at 15 28 01

mobile:
Screenshot 2024-03-04 at 15 27 44

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/components/CardWithFeaturedItems/CardWithFeaturedItems.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+type CardWithFeaturedItemsProps = {
+  heading: string
+  imgSrc: string
+  alt: string
+  href?: string | null
+  dataTestId?: string
+  seeMoreText: string
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore make web strict
+  featuredItems: GetFrontpageQuery['getFrontpage']['lifeEvents'][0]['featured']
```
Comment (splicebison) [inline apps/web/components/CardWithFeaturedItems/CardWithFeaturedItems.tsx:16]:
```suggestion
  featuredItems: LifeEventPage['featured']
```

We should consider here relying also on the `LifeEventPage` type or, alternatively, use the `Featured` type directly from `'@island.is/web/graphql/schema'`

File: apps/web/components/LifeEventsSection/LifeEventsSection.tsx [hunk 2]
```diff
@@ -1,31 +1,83 @@
...
       <GridContainer>
-        <Text variant="h3" as="h2" id={headingId}>
-          {heading}
-        </Text>
+        <Box display="flex" flexDirection="row" justifyContent="spaceBetween">
+          <Text variant="h2" as="h2" id={headingId}>
+            {heading}
+          </Text>
+          <Box display={['none', 'none', 'none', 'block']}>
+            <Link href={'/lifsvidburdir'} skipTab>
```
Comment (splicebison) [inline apps/web/components/LifeEventsSection/LifeEventsSection.tsx:39]: This should use the link resolver for internationalization.

File: apps/web/components/LifeEventsSection/LifeEventsSection.tsx [hunk 3]
```diff
@@ -34,27 +86,47 @@ export const LifeEventsSection = ({
...
+      <Hidden above="md">
+        <GridContainer>
+          <Box
+            display="flex"
+            width="full"
+            justifyContent="center"
+            alignItems="center"
+            paddingY={[3, 3, 3, 0]}
+          >
+            <Link skipTab href={'/lifsvidburdir'}>
```
Comment (splicebison) [inline apps/web/components/LifeEventsSection/LifeEventsSection.tsx:117]: Also need link resolver here

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Pull request closed without merging**